src/pages/index.js

```diff
@@ -105,9 +106,9 @@ function IndexPage() {
             <div className="col-sm-12 our_mission_content text-center">
               <blockquote>
                 <p>
-                  Through training, mentorship, building our open source projects, and opportunities
-                  industry experts, we ensure that our troops gain the best knowledge possible
-                  regards to learning software skills that can get them employed.
+                  Through training, mentorship, building new features or enhancing our open-source projects, and opportunities
+                  to interact with industry experts, our goal is to ensure that our troops gain the best knowledge possible
+                  regarding to learning software skills that help them get gainfully employed in the tech sector.
```

Comment:
I think this section 
> "help them get gainfully employed"

 would probably sound more natural as "obtain gainful employment".

Reply:
Through training, mentorship, working with our open-source projects, and opportunities
to interact with industry experts, our goal is to ensure that our troops gain the best knowledge possible
while learning software skills that help them become gainfully employed in the technology sector.